Index: mingw-ocaml-git/build/ocaml/asmrun/i386.S
===================================================================
--- mingw-ocaml-git.orig/build/ocaml/asmrun/i386.S	2014-02-16 11:01:12.002027930 +0900
+++ mingw-ocaml-git/build/ocaml/asmrun/i386.S	2014-02-16 11:01:44.010860809 +0900
@@ -75,6 +75,9 @@
         movl %esp, %ebp; \
         call mcount; \
         popl %ebp; CFI_ADJUST(-4)
+#elif defined(SYS_mingw)
+#define PROFILE_CAML
+#define PROFILE_C
 #elif defined(SYS_bsd_elf)
 #define PROFILE_CAML \
         pushl %ebp; CFI_ADJUST(4); \
